src/object.rs

```diff
@@ -1228,9 +1235,9 @@ unsafe impl IsClassFor for ObjectClass {
 unsafe impl Send for ObjectClass {}
 unsafe impl Sync for ObjectClass {}
 
-pub struct WeakRef<T: IsA<Object> + ?Sized>(Box<gobject_ffi::GWeakRef>, PhantomData<*const T>);
+pub struct WeakRef<T: IsA<Object>>(Box<gobject_ffi::GWeakRef>, PhantomData<*const T>);
```

Comment:
This change really "require fewer" ? 😉
Or unsized types better not used now?

Reply:
Indeed that one is not really fewer :) It can't really work right now for unsized/dynamically sized types (e.g. `IsA` is not object-safe, and various other issues). I have fixing that on my list but that's still a longer way.

Reply:
Updated the commit message with that